.github/scripts/tests/tests.sh

```diff
@@ -165,34 +166,54 @@ run_tests() {
   echo "---------------------------------"
   echo "Run tests"
   echo "---------------------------------"
-  ( cd $GIT_WORKSPACE && make integrationtest K8SAPISERVERHOST=$(oc whoami --show-server) DSPANAMESPACE=${DSPA_NAMESPACE} DSPAPATH=${DSPA_PATH} )
+  ( cd $GIT_WORKSPACE && make integrationtest K8SAPISERVERHOST=$(kubectl whoami --show-server) DSPANAMESPACE=${DSPA_NAMESPACE} DSPAPATH=${DSPA_PATH} )
 }
 
 run_tests_dspa_external_connections() {
   echo "---------------------------------"
   echo "Run tests for DSPA with External Connections"
   echo "---------------------------------"
-  ( cd $GIT_WORKSPACE && make integrationtest K8SAPISERVERHOST=$(oc whoami --show-server) DSPANAMESPACE=${DSPA_EXTERNAL_NAMESPACE} DSPAPATH=${DSPA_EXTERNAL_PATH} )
+  ( cd $GIT_WORKSPACE && make integrationtest K8SAPISERVERHOST=$(kubectl whoami --show-server) DSPANAMESPACE=${DSPA_EXTERNAL_NAMESPACE} DSPAPATH=${DSPA_EXTERNAL_PATH} )
 }
 
-clean_up() {
+undeploy_kind_resources() {
   echo "---------------------------------"
   echo "Clean up"
   echo "---------------------------------"
   ( cd $GIT_WORKSPACE && make undeploy-kind )
 }
 
+remove_namespace_created_for_rhoai() {
```

Comment:
can we get a comment here on why this is needed? and a log output as well? 

I'm guessing this is to cleanup old artifacts from a previous run?

Reply:
> I'm guessing this is to cleanup old artifacts from a previous run?

There is a special task in the tracking document about the cleanup. We can discuss about that when we start working on the cleanup. Example:
* Should we do on the begin or in the end of the function?
* What should be default?
* Should we set a parameter or not?

For now, it is on the beginning to ensure that we won't have any flaky test.

Reply:
my suggestion is to include more description in the code, and add a log message like an echo statement explaining the step or some feedback for the user

Reply:
Done